### ISSUE TITLE
[4.0] Icons in scene tree overlap node names

### ISSUE THREAD

**Issue by git2013vb** [godotengine/godot#57140]
### Godot version

4.0alpha 1

### System information

Debian 11

### Issue description

When I reduce the horizontal space to the min in scene tree I still see the icons overlap node name even using horizontal scroll bar.
see picture

![image](https://user-images.githubusercontent.com/13235119/150839594-c7a7c651-787c-40f9-9708-acc4284b22e5.png)

I was expecting to see the last control node entirely.

### Steps to reproduce

Create tree as in picture the scroll horizontal bar

### Minimal reproduction project

NA

**Comment by KoBeWi** [comment]
Related to #28823 (the other one could be maybe closed, as it's older and looks a bit different)

**Comment by git2013vb** [comment]
> Related to #28823 (the other one could be maybe closed, as it's older and looks a bit different)

Right. I think the best solution  is to keep max one icon and when there are more open a po-pup context menu using vertical 3 dots. Then fix the current issue ;)
